test/testdata/intoto/pipeline-output-image.json

```diff
@@ -42,7 +45,12 @@
                     ],
                     "invocation": {
                         "configSource": {},
-                        "parameters": {}
+                        "parameters": {},
+                        "environment": {
+                            "annotations": {
+                                "pipeline.tekton.dev/release": "9221ac3-dirty"
```

Comment:
Is this value hard-coded `9221ac3-dirty`? Just checking. Probably don't want this to change if we update the installation of Tekton pipelines in CI.

Reply:
You're right... I think I'll have to delete this attribute from the annotations before the call to Diff.

Reply:
You can have cmp.Diff ignore fields with this! https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#IgnoreFields

Reply:
Done